rest-api-spec/src/main/resources/rest-api-spec/test/index/110_constant_keyword.yml

```diff
@@ -0,0 +1,62 @@
+---
+setup:
+  - do:
+      indices.create:
+        index: test
+        body:
+          mappings:
+            properties:
+              genre:
+                type : "constant_keyword"
+                "value" : "Romantic comedy"
```

Comment:
Nit: does this need to be a quoted "value"? Looks inconsistent with type.

Reply:
Yes, it defines the all documents must have the same  field value.